src/test_driver/tizen/README.md

```diff
@@ -12,7 +12,7 @@ image from hub.docker.com or build it locally using the provided Dockerfile in
 
 ```sh
 # Pull the image from hub.docker.com
-docker pull ghcr.io/project-chip/chip-build-tizen-qemu:latest
```

Comment:
This seems painful to keep up to date. latest might be easier in this case (even if not always correct)

Reply:
Run the script, we'll require everyone does it. It does it automatically.

Reply:
And eventually, as I said, we will not allow anyone to edit these, this will be done by a bot here: https://github.com/project-chip/connectedhomeip/pull/28001